site/docs/configuration/expected-outputs/python.md

```diff
@@ -39,19 +39,20 @@ assert:
             'pass': True,
             'score': 0.5,
           }
-      else:
-          return {
-            'pass': False,
-            'score': 0,
-          }
+      return {
+        'pass': False,
+        'score': 0,
+      }
 ```
 
 ## Using test context
 
 A `context` object is available in the Python function. Here is its type definition:
 
 ```py
-class AssertContext:
+from typing import Dict, TypedDict, Union
+
+class AssertContext(TypedDict):
```

Comment:
Shouldn't `vars` allow for `dict` too? Was that what was meant by `object`?

Reply:
Also, it looks like `test` (also a `dict`) should be here too

Reply:
Yeah, I think you're right on both counts

Reply:
Okay, implemented them, this is ready for review now